<ui version="4.0" >
 <class>Prototyper</class>
 <widget class="QDialog" name="Prototyper" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Prototyper</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images.qrc" >
    <normaloff>:/icons/images/NetworkEditor-icon.png</normaloff>:/icons/images/NetworkEditor-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="playStatus" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../images.qrc" >:/icons/images/clamlogo.png</pixmap>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../images.qrc" >:/icons/images/NetworkEditor-icon.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="PlaybackIndicator" />
      </item>
      <item>
       <widget class="QLabel" name="BackendIndicator" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="PlayButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Play</string>
       </property>
       <property name="icon" >
        <iconset resource="../images.qrc" >
         <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PauseButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Stop</string>
       </property>
       <property name="icon" >
        <iconset resource="../images.qrc" >
         <normaloff>:/icons/images/player_pause.png</normaloff>:/icons/images/player_pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Stop</string>
       </property>
       <property name="icon" >
        <iconset resource="../images.qrc" >
         <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="CloseButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Exit</string>
     </property>
     <property name="icon" >
      <iconset resource="../images.qrc" >
       <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>PlayButton</tabstop>
  <tabstop>StopButton</tabstop>
  <tabstop>CloseButton</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>Prototyper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>231</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel" >
     <x>240</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
